### DISCUSSION TITLE
DistroTube has some interesting stuff to say about AppImage

### DISCUSSION THREAD

**Discussion by grd** [helloSystem/hello#348] [Ideas]
[DistroTube AppImage link](https://youtu.be/Yk6kaaoTKmE)

I like that guy. He likes to keep on talking and he knows quite a lot. He likes AppImage more than snap and flatpak. And then he shows the link to where most AppImages are stored and after that he shows an App for downloading Apps. This is good stuff and I like it a lot. Of course it is not polished. But this is how it should be in helloSystem. A nice app where you can download your app from all over the world.

I can be wrong but I like it.

**Comment by probonopd** [comment]
Thanks @grd.

It's interesting that AppImage was _specifically_ made to work like an .exe or .dmg that you can download straight from an application author's website so that you won't need app stores nor package managers, yet Linux people seem to make app stores and package managers around them all the time.
It's probably because that's how Linux users are trained to "manage" software, whereas I am looking to recreate the simplicity known from the original Macintosh (1984).

**Reply by grd** [reply]
I agree. It is a good design, that is timeless.

**Comment by grd** [comment]
This is funny. The guy who made this video now made a new one. This time it's about Updating AppImages. See [link](https://youtu.be/V3i1R8qrGAg).

As always it is not perfect and it hangs somewhere but it's funny. I like that guy.
